src/explore-education-statistics-admin/src/services/apiDataSetVersionService.ts

```diff
@@ -154,6 +156,17 @@ const apiDataSetVersionService = {
       data,
     );
   },
+  getChanges(dataSetVersionId: string): Promise<ApiDataSetVersionChanges> {
+    return client.get(
+      `/public-data/data-set-versions/${dataSetVersionId}/changes`,
+    );
+  },
+  updateNotes(data: {
+    dataSetVersionId: string;
+    notes: string;
+  }): Promise<ApiDataSetVersion> {
+    return client.patch(`/public-data/data-set-versions`, data);
```

Comment:
I'm really not sure what Jack was doing here, but the URL for this isn't correct! I'm going to push a change to fix this

Reply:
Thanks!